src/ssr.rs

```diff
@@ -1,156 +1,187 @@
-use std::collections::HashMap;
-
-#[derive(Clone, Debug, PartialEq)]
-pub struct Ssr<'a> {
-    // TODO: Check if better Box<str> instead of String
-    source: String,
-    entry_point: &'a str,
+pub struct SSREnvironment {
+    source_code: String,
+    entry_point: String,
```

Comment:
Why shouldn't `entry_point` and `root_export` be a `&'a str`?

Reply:
Again, here I went for a copy just so the user can delete the original string and keep the SSREnvironment. Might be better to make these all `'a` if we are going to make `source_code` a `'a`.